src/type-translator.ts

```diff
@@ -164,12 +164,24 @@ export class TypeTranslator {
    * - TypeChecker.typeToString translates Array as T[].
    * - TypeChecker.symbolToString emits types without their namespace,
    *   and doesn't let you pass the flag to control that.
+   * @param useFqn whether to scope the name using its fully qualified name.
    */
-  public symbolToString(sym: ts.Symbol): string {
+  public symbolToString(sym: ts.Symbol, useFqn: boolean): string {
```

Comment:
Would it make sense to default this to true?  It seems you only want it off in one special case.